### PR TITLE
E2E for Scenario 3.1.1

### PR DESCRIPTION
**Pull request by sergiishamrai-okta**
What is done:

- E2E for Scenario 3.1.1
- Adding and Removing Okta user for scenarios 3.X
- Removing newly registered users in scenarios 4.X
- More logs

New Env variables are required:

- OKTA_CLIENT_CLIENTID
- OKTA_CLIENT_ORGURL
- OKTA_CLIENT_TOKEN

### CODE REVIEW

**Review by VitaliiTytarenko-okta — APPROVED** [review]
👍

File: direct-auth-samples/pom.xml [hunk 1]
```diff
@@ -167,6 +167,24 @@
             <version>5.3.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (arvindkrishnakumar-okta) [inline direct-auth-samples/pom.xml:170]: Can we add a property placeholder for `4.0.0` version?

File: direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/PasswordRecovery.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import pages.*;
+
+public class PasswordRecovery extends CucumberRoot {
+
+	private static final int RETRY_COUNT = 5; //TODO Should be in config
```
Comment (arvindkrishnakumar-okta) [inline direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/PasswordRecovery.java:31]: Can you create a followup/backlog JIRA item for this?

**Review by arvindkrishnakumar-okta — APPROVED** [review]
Minor changes suggested.

File: direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/PasswordRecovery.java (hunk 2 above)
Comment (sergiishamrai-okta) [reply direct-auth-samples/src/test/java/info/seleniumcucumber/userStepDefinitions/PasswordRecovery.java:31]: Int ref: [OKTA-401435](https://oktainc.atlassian.net/browse/OKTA-401435)

**Pull request merged**